### PR TITLE
Add trendy-terminal 0.0.5

### PR DESCRIPTION
**Pull request by pyjarrett**

### CODE REVIEW

File: index/tr/trendy_terminal/trendy_terminal-0.0.5.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+authors = ["Paul Jarrett"]
+maintainers = ["Paul Jarrett <jarrett.paul.young@gmail.com>"]
+maintainers-logins = ["pyjarrett"]
+licenses = "Apache-2.0"
+tags = ["terminal", "command-line", "console", "vt100"]
+
+project-files = ["trendy_terminal.gpr"]
+
+[gpr-externals]
+Mode = ["debug", "optimize"]
```
Comment (mosteo) [inline index/tr/trendy_terminal/trendy_terminal-0.0.5.toml:15]: `Mode` is probably too prone to clashes with other crates. The current advice is to prefix with the crate name.

File: index/tr/trendy_terminal/trendy_terminal-0.0.5.toml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+[gpr-set-externals.'case(os)']
+windows = { Trendy_Terminal_Platform = "windows" }
+linux = { Trendy_Terminal_Platform = "linux" }
+macos = { Trendy_Terminal_Platform = "macos" }
+
+[available.'case(os)']
+linux = true
+windows = true
+macos = true
```
Comment (mosteo) [inline index/tr/trendy_terminal/trendy_terminal-0.0.5.toml:25]: Default availability is true, so this can be removed.

**Review by mosteo — CHANGES_REQUESTED** [review]

File: index/tr/trendy_terminal/trendy_terminal-0.0.5.toml (hunk 1 above)
Comment (pyjarrett) [reply index/tr/trendy_terminal/trendy_terminal-0.0.5.toml:15]: I think this got auto-generated a long time ago and I've never bothered with it.

**Comment by pyjarrett** [comment]
Should be fixed now

**Review by mosteo — APPROVED** [review]

**Pull request merged**